src/main/java/fanny/FannyException.java

```diff
@@ -0,0 +1,16 @@
+package fanny;
+
+/**
+ * Represents an exception unique to the Fanny application.
+ */
+public class FannyException extends Exception {
```

Comment:
Good naming of class to specify Exception that is specific to Fanny! 😃

Reply:
Thank you!